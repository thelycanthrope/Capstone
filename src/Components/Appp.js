import React from 'react'
import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
import TopViewed from '../pages/TopViewed';
import AllProductsPage from '../pages/Products/AllProductsPage';
import UserDetails from '../pages/UserDetails';
import Home from '../pages/Home';
import SignIn from '../pages/SignIn/SignIn';
import Footer from './Footer';

import AddProduct from '../pages/AddProducts/AddProduct';
import ProductDetails from '../pages/Products/ProductDetails';
import Update from '../pages/Products/Update';
import Register from '../pages/SignIn/Register';


function Appp() {
    return (
        <div >
            
            <Router>
            
               
            <Switch>
                <Route exact path="/home" component={Home} />
               
                <Route exact path="/TopViewed" component={TopViewed} />
                <Route exact path='/UserDetails' component={UserDetails}/>
                <Route exact path='/signIn' component={SignIn}/>
                <Route exact path='/products' component={AllProductsPage}/>
                <Route exact path='/addproduct' component={AddProduct}/>
                <Route path="/products/" component={ProductDetails} ></Route>
                <Route exact path="/update" component={Update}></Route>
                <Route path='/register' component={Register}></Route>
                <Redirect from="/" to='/home' />
            </Switch>
            </Router>
           
            <Footer/>
        </div>
    )
}

export default Appp
