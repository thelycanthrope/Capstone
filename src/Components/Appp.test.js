import Appp from './Appp'
import {shallow,configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';


configure({adapter: new Adapter()});
describe ('Appp testing', () =>{
    it('renders three components', () => {
        const wrapper = shallow(<Appp />);
        expect(wrapper).toMatchSnapshot()
      });
})