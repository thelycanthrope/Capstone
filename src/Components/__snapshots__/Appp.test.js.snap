// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appp testing renders three components 1`] = `ShallowWrapper {}`;
